src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -91,7 +91,7 @@ class PalantirDockerPlugin implements Plugin<Project> {
                 into dockerDir
             }
 
-            List<String> buildCommandLine = ['docker', 'build']
+            List<String> buildCommandLine = ['docker', 'build', '--quiet']
```

Comment:
Can we inspect whether Gradle was called with --info or --debug and set this accordingly?